packages/module-web/src/client/core/schema/components/header/Header.tsx

```
@@ -54,15 +55,30 @@ const InternalHeader = (props: HeaderProps) => {
+              isHeaderEnabled: true,
+              fieldSchema: parentFieldSchema,
+              title,
+              enabledSharePage,
+            };
+            return (
+              <PageExtendComponentProvider {...componentProps}>
+                <SchemaComponent schema={schema} />
+              </PageExtendComponentProvider>
+            );
+          })}
+        </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)